src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -61,59 +64,112 @@ const DeleteWorkspaceModal = ({ workspace, workspace: { workspace: { name, bucke
-      h(TextInput, {
-        id: 'delete-workspace-confirmation',
-        placeholder: 'Delete Workspace',
-        value: deleteConfirmation,
-        onChange: setDeleteConfirmation
-      })
-    ]),
-    (deleting || loading) && spinnerOverlay
-  ])
+  const renderAzureCleanupModal = () => {
+    return h(Modal, {
+      title: span({ style: { display: 'flex', alignItems: 'center' } }, [
+        icon('warning-standard', { size: 24, color: colors.warning() }),
+        span({ style: { marginLeft: '1ch' } }, ['Delete workspace resources'])
+      ]),
+      onDismiss,
+      okButton: h(ButtonPrimary, {
+        disabled: isDeleteDisabledFromResources,
+        onClick: deleteWorkspaceResources
+      }, ['Delete all resources']),
+      styles: { modal: { background: colors.warning(0.1) } }
+    },
+    [
+      isDeleteDisabledFromResources && !deletingResources && div([
+        p(['This workspace has resources that are not deletable.']),
+        p(['If the resource is provisioning or deleting, try again in a few minutes. Please reach out to support@terra.bio for assistance.']),
+        ul([
+          workspaceResources.nonDeleteableApps.map(app => li({ key: app.appName }, [app.appName, ` (${app.status.toLowerCase()})`])),
+          workspaceResources.nonDeleteableRuntimes.map(
+            runtime => li({ key: runtime.runtimeName }, [runtime.runtimeName, ` (${runtime.status.toLowerCase()})`]))
+        ])
+      ]),
+      (!isDeleteDisabledFromResources || deletingResources) && div([
+        p(['This workspace cannot be deleted because of the following running cloud resources:']),
+        ul([
+          workspaceResources.apps.map(app => li({ key: app.appName }, [app.appName])),
+          workspaceResources.runtimes.map(runtime => li({ key: runtime.runtimeName }, [runtime.runtimeName]))
+        ]),
+        p(['These resources must be deleted before the workspace can be deleted.']),
+        p(['It may take several minutes to delete all of the cloud resources in this workspace. Please do not close this window']),
+        p([strong(['This cannot be undone.'])])
+
+      ]),
+      (deletingResources || loading) && spinnerOverlay
+    ]
+    )
+  }
+
+  const renderWorkspaceDeletionModal = () => {
+    return h(Modal, {
+      title: span({ style: { display: 'flex', alignItems: 'center' } }, [
+        icon('warning-standard', { size: 24, color: colors.warning() }),
+        span({ style: { marginLeft: '1ch' } }, ['Delete workspace'])
+      ]),
+      onDismiss,
+      okButton: h(ButtonPrimary, {
+        disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromResources,
+        onClick: deleteWorkspace,
+        tooltip: Utils.cond(
+          [isDeleteDisabledFromResources && isGoogleWorkspace(workspace), () => 'You must ensure all apps in this workspace are deletable'],
+          [isDeleteDisabledFromResources && isAzureWorkspace(workspace), () => 'This workspace cannot be deleted'],
```

Comment:
I am surprised that I am not seeing the tooltip text in this case. Is something wrong with this code?

![image](https://user-images.githubusercontent.com/484484/221607451-dac553f1-bd8c-44f6-95ec-87d71bc4c173.png)

Reply:
Oh, I see that it's actually a different dialog. I think we should add the tooltip text up there.